test/integration/multikueue/multikueue_test.go

```diff
@@ -79,37 +94,161 @@ var _ = ginkgo.Describe("Multikueue", func() {
 		}
 
 		gomega.Expect(managerCluster.client.Create(managerCluster.ctx, managerMultikueueSecret)).To(gomega.Succeed())
+
+		managerMultiKueueConfig = &kueue.MultiKueueConfig{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "multikueueconfig",
+			},
+			Spec: kueue.MultiKueueConfigSpec{
+				Clusters: []kueue.MultiKueueCluster{
+					{
+						Name: "worker1",
+						KubeconfigRef: kueue.KubeconfigRef{
+							SecretName:      "multikueue",
+							SecretNamespace: managerNs.Name,
+							ConfigKey:       "worker1.kubeconfig",
+						},
+					},
+					{
+						Name: "worker2",
+						KubeconfigRef: kueue.KubeconfigRef{
+							SecretName:      "multikueue",
+							SecretNamespace: managerNs.Name,
+							ConfigKey:       "worker2.kubeconfig",
+						},
+					},
+				},
+			},
+		}
+		gomega.Expect(managerCluster.client.Create(managerCluster.ctx, managerMultiKueueConfig)).Should(gomega.Succeed())
+
+		managerAc = utiltesting.MakeAdmissionCheck("ac1").
```

Comment:
Does this admission check play the role of the MultiKueue admission check from the [KEP](https://github.com/mwielgus/kueue/blob/e8c208c4c6d9d4e988049392de35c1105222ccd8/keps/693-multikueue/README.md#summary)? `Introduce an new AdmissionCheck (called MultiKueue) with dedicated API and controller that will provide multi-cluster capabilities to Kueue.`?

Should we call it `MultiKueue` in that case?